### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by trombik**
add simple tests to see a log file is rotated as expected. in the development, it was found that the role failed on `ubuntu` due to the lack of `su` directive in the default configuration. the issues was also fixed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,30 +16,31 @@ None
 | logrotate\_default\_dateext | the default value of `dateext` in `logrotate.conf` | true |
 | logrotate\_default\_dateformat | the default value of `dateformat` in `logrotate.conf` | .%Y%m%d |
 | logrotate\_default\_freq | the default value of how often rotate the logs in `logrotate.conf` | daily |
+| `logrotate_default_su` | | `{{ __logrotate_default_su }}` |
```
Comment (mitsururike) [inline README.md]: no description?

File: library/logrotate.py [hunk 2]
```diff
@@ -160,11 +160,32 @@ def validate_config(module):
...
+def get_default_config_path(module):
+    """Look for the default configuration and return the first one found"""
+    locations = [
+        # Linux
+        '/etc/logrotate.conf',
+        # FreeBSD
+        '/usr/local/etc/logrotate.conf'
+        ]
+    found = ''
+    for path in locations:
```
Comment (mitsururike) [inline library/logrotate.py:181]: does FreeBSD's logrotate read /etc/logrotate.conf?

File: tests/integration/example/spec/default_spec.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+    result = current_server.ssh_exec("sudo logrotate -f -v #{ conf_path }")
+    expect(result).not_to match(/^\s+log #{ log_file_regexp }.*skipping$/)
+    expect(result).to match(/^reading config file foo$/)
+    expect(result).to match(/^rotating pattern:\s+#{ log_file_regexp }$/)
+    expect(result).to match(/^considering log #{ log_file_regexp }\n\s+log needs rotating$/)
+    expect(result).to match(/^renaming #{ log_file_regexp } to #{ log_file_regexp}\.1$/)
+  end
+
+  it "creates /var/log/foo.log.1" do
```
Comment (mitsururike) [inline tests/integration/example/spec/default_spec.rb]: use #{log_file}

**Review by mitsururike — COMMENTED** [review]
added comments on some lines

File: library/logrotate.py (hunk 2 above)
Comment (trombik) [reply library/logrotate.py:181]: no, it does not. i was looking for get the value of `ansible_os_family` within a module but failed. the logic is just guessing OS by the location.

File: README.md (hunk 1 above)
Comment (trombik) [reply README.md]: fixed in 91307e6

File: tests/integration/example/spec/default_spec.rb (hunk 3 above)
Comment (trombik) [reply tests/integration/example/spec/default_spec.rb]: fixed in 91307e6

**Review by mitsururike — APPROVED** [review]

**Pull request merged**